ocpkg

```diff
@@ -630,6 +636,77 @@ rm -rf master.tar.gz cogutil-master/
 cd $CURRENT_DIR
 }
 
+# Install notebooks
+install_notebooks(){
+MESSAGE="Installing notebooks...." ; message
+
+
+#####INSTALLING pip3 AND jupyter notebook
+cd
+sudo apt-get install python3-pip
+python3 -m pip install --upgrade pip
+python3 -m pip install ipython jupyter
+
+#####INSTALLING GUILE KERNEL#####
+#check if site folder exits
+if [ ! -d "/usr/local/share/guile/site" ]
+then
+    cd /usr/local/share/guile/
+    sudo mkdir site
+    cd
+fi
+
+#Install ZeroMQ library
+##Clean up remenants of previous installations##
+rm -rf zeromq-4.2.1*
+wget https://github.com/zeromq/libzmq/releases/download/v4.2.1/zeromq-4.2.1.tar.gz
+tar xvf zeromq-4.2.1.tar.gz
+cd zeromq-4.2.1/
+./configure
+make
+sudo make install
+cd
+
+#Install guile-json library
+##Clean up remenants of previous installations##
+rm -rf guile-json-0.6.0*
+wget http://download.savannah.gnu.org/releases/guile-json/guile-json-0.6.0.tar.gz
```

Comment:
why not use https://packages.ubuntu.com/xenial/guile-json?

Reply:
This is from the guile kernel installation at https://github.com/jerry40/guile-kernel, but I did go ahead and test the one in the link suggested and the files do not get to the paths we need. You will get a "no code for json" error.

Reply:
not exactly, but I did move the Json files and directory that are created by the guile-json (ubuntu package) to the preferred path at /usr/local but the guile kernel constantly dies & restarts with error - 
json/builder.scm: 131:4 0(json-build-array (("msg_id" . "7b136e32ddcbe2e226...") ...) ...)
json/builder.scm:131:4: In procedure json-build-array:
Throw to key 'json-invalid' with args '()'. 

So the Ubuntu package version of guile-json just won't do.